### DISCUSSION TITLE
When using Spatie translatable how can i make sure every language will be filled ?

### DISCUSSION THREAD

**Discussion by bumpyy** [filamentphp/filament#6913] [Help] [answered]
So basically, I want to make sure every language will be filled with the initial value when creating the data regardless the local i choose, then i can just edit it.

example: when i create new data and  im on 'Indonesian', the 'English" version will be empty, it will be better if the "English" and other local to be also filled with the same data.

**Comment by pboivin** [comment, marked as answer]
I'm curious to know what's the use-case for this? Are you trying to have a fallback value instead of an empty value? Maybe this can be useful :
https://spatie.be/docs/laravel-translatable/v6/basic-usage/handling-missing-translations#content-falling-back-to-any-locale

**Reply by bumpyy** [reply]
That's the spec my client gave me (more like "revision"), i dunno if they want to do something with the data in the database that need all localization to be filled, but i think this will do for the frontend side.

Thanks.
